src/main/resources/subject-details/translation.json

```diff
@@ -49,7 +49,8 @@
     "cy": {
       "subjectFullName": {
         "required": "Nodwch enw llawn",
-        "containsMarkdownLink": "The data entered is not valid for a full name. Link markup characters are not allowed in this field (WELSH)"
+        "containsMarkdownLink": "The data entered is not valid for a full name. Link markup characters are not allowed in this field (WELSH)",
+        "containsInvalidCharacters": "Full name must not include HTML (WELSH)"
```

Comment:
Have these been added to the corresponding Welsh ticket?

Reply:
These will be when QA approves the error messages so that we can be sure there will be no other changes to the English counterparts that need a translation